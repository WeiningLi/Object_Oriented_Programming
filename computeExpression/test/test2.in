x y + z - 6 * 2 / NEG 5 x + y * + ABS done
copy
set y 1
eval
set x 12
eval
copy
print
eval
unset x
copy
print
eval
unset y
copy

