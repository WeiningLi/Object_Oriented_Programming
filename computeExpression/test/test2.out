|(-((((x + y) - z) * 6) / 2) + ((5 + x) * y))|
|(-((((x + y) - z) * 6) / 2) + ((5 + x) * y))|
|(-((((1 + y) - z) * 6) / 2) + ((5 + 1) * y))|
y has no value.
x has no value.
z has no value.
|(-((((12 + 1) - z) * 6) / 2) + ((5 + 12) * 1))|
|(-((((1 + 1) - z) * 6) / 2) + ((5 + 1) * 1))|
z has no value.
|(-((((12 + 1) - z) * 6) / 2) + ((5 + 12) * 1))|
z has no value.
|(-((((x + 1) - z) * 6) / 2) + ((5 + x) * 1))|
|(-((((1 + 1) - z) * 6) / 2) + ((5 + 1) * 1))|
z has no value.
|(-((((x + 1) - z) * 6) / 2) + ((5 + x) * 1))|
x has no value.
|(-((((x + y) - z) * 6) / 2) + ((5 + x) * y))|
|(-((((1 + y) - z) * 6) / 2) + ((5 + 1) * y))|
y has no value.
